modules/crud/sql/describe.go

```diff
@@ -41,9 +70,30 @@ func (s *SQL) getDescribeDetails(ctx context.Context, col string) ([]utils.Field
 	return result, nil
 }
 
-func (s *SQL) getForeignKeyDetails(ctx context.Context, project, col string) ([]utils.ForeignKeysType, error) {
-	fmt.Println("Sql", project, col)
-	rows, err := s.client.Queryx("select TABLE_NAME, COLUMN_NAME, CONSTRAINT_NAME, REFERENCED_TABLE_NAME, REFERENCED_COLUMN_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE REFERENCED_TABLE_SCHEMA = '" + project + "' and TABLE_NAME = '" + col + "'")
+func (s *SQL) getForeignKeyDetails(ctx context.Context, project, col, dbType string) ([]utils.ForeignKeysType, error) {
+	queryString := ""
+	if utils.DBType(dbType) == utils.MySQL {
+		queryString = "select TABLE_NAME, COLUMN_NAME, CONSTRAINT_NAME, REFERENCED_TABLE_NAME, REFERENCED_COLUMN_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE REFERENCED_TABLE_SCHEMA = '" + project + "' and TABLE_NAME = '" + col + "'"
+	} else {
+		queryString = `SELECT
+		tc.table_name AS "TABLE_NAME", 
+		kcu.column_name AS "COLUMN_NAME", 
+		tc.constraint_name AS "CONSTRAINT_NAME", 
+		ccu.table_name AS "REFERENCED_TABLE_NAME",
+		ccu.column_name AS "REFERENCED_COLUMN_NAME"
+	FROM 
+		information_schema.table_constraints AS tc 
+		JOIN information_schema.key_column_usage AS kcu
+		  ON tc.constraint_name = kcu.constraint_name
+		  AND tc.table_schema = kcu.table_schema
+		JOIN information_schema.constraint_column_usage AS ccu
+		  ON ccu.constraint_name = tc.constraint_name
+		  AND ccu.table_schema = tc.table_schema
+	WHERE tc.constraint_type = 'FOREIGN KEY' AND tc.table_name='` + col + `'
```

Comment:
Take `col` as `:col`. Pass argument to the `Queryx` function below

Reply:
same here

Reply:
done

Reply:
This works?

Reply:
yes